server/command_hooks.go

```diff
@@ -522,6 +534,50 @@ func (p *Plugin) crash() {
 	_ = 1 / y
 }
 
+func (p *Plugin) executeCommandMentions(args *model.CommandArgs) *model.CommandResponse {
+	message := "The command `" + args.Command + "` contains the following different mentions.\n"
```

Comment:
Does it make sense to make use of `text/template` here and get rid of the string concatenation?

Reply:
Did not know about that package, I'll make use of it, thanks!

Reply:
3/5, but importing `text/template` just to get rid of string concatenation seems like overkill.

Reply:
I first thought of that, but nevertheless tried to implement it with a template to see how it looked like. However, I have now another counterargument: the goal of this command is to showcase how to use the new structures in the code itself, and inlining the access to the mentions structures inside the template, makes it harder to read.
So I would also leave it as it is right now. Further thoughts, @hanzei?

Reply:
This is powerful (thanks for the tip!) but hides all the important details in this specific case:
```go
	const message = `
The command` + "`{{.Command}}`" + `contains the following different mentions.

### Mentions to users in the team
{{if not .UserMentions}}
_There are no mentions to users in the team in your command_.
{{else}}
| User name | ID |
|-----------|----|
{{- range $name, $id := .UserMentions}}
|@{{$name}}|{{$id}}|
{{end}}
{{end}}

### Mentions to public channels
{{if not .ChannelMentions}}
_There are no mentions to public channels in your command_.
{{else}}
| Channel name | ID |
|--------------|----|
{{range $name, $id := .ChannelMentions -}}
|~{{$name}}|{{$id}}|
{{end}}
{{end}}
`
```

Reply:
Makes sense :+1: I'm fine leaving it as it is